### PR TITLE
Cards vue G+H

### PR DESCRIPTION
**Pull request by chosta**
More cards

### CODE REVIEW

File: src/cards/base/GHGProducingBacteria.ts [hunk 1]
```diff
@@ -56,4 +59,20 @@ export class GHGProducingBacteria implements IActionCard, IProjectCard, IResourc
...
       return orOptions;
     }
+    public metadata: CardMetadata = {
+      description: 'Requires 4% oxygen.',
+      cardNumber: '034',
+      requirements: CardRequirements.builder((b) => b.oxygen(4)),
+      renderData: CardRenderer.builder((b) => {
+        b.effectBox((eb) => {
+          eb.empty().startAction.microbes(1);
+          eb.description('Action: Add 1 Microbe to this card.');
```
Comment (kberg) [inline src/cards/base/GHGProducingBacteria.ts:69]: A thought: if every Effect Box has a decsription, you don't really need eb.description. This could be written instead as

```
b.effectBox('Action: Add 1 Microbe to this card.', (eb) => {
  eb.empty().startAction.microbes(1);
});
```

File: src/cards/base/GreatEscarpmentConsortium.ts [hunk 2]
```diff
@@ -19,4 +23,15 @@ export class GreatEscarpmentConsortium implements IProjectCard {
...
+    public metadata: CardMetadata = {
+      cardNumber: '061',
+      requirements: CardRequirements.builder((b) => b.production(Resources.STEEL)),
+      renderData: CardRenderer.builder((b) => {
+        b.productionBox((pb) => {
+          pb.minus().steel(-1).any.br;
+          pb.plus().steel(1);
+        });
+      }),
+      description: 'Requires that you have steel production. Decrease any steel production 1 step and increase your own 1 step.',
```
Comment (kberg) [inline src/cards/base/GreatEscarpmentConsortium.ts:35]: Should steel be capitalized?

File: src/cards/prelude/HousePrinting.ts [hunk 3]
```diff
@@ -18,4 +20,12 @@ export class HousePrinting implements IProjectCard {
     public getVictoryPoints() {
       return 1;
     }
+    public metadata: CardMetadata = {
+      cardNumber: 'P36',
+      renderData: CardRenderer.builder((b) => {
+        b.productionBox((pb) => pb.steel(1));
+      }),
+      description: 'Increase your steel production 1 step.',
```
Comment (kberg) [inline src/cards/prelude/HousePrinting.ts:28]: Same?

File: src/components/card/CardRenderTileComponent.ts [hunk 4]
```diff
@@ -47,6 +47,8 @@ export const CardRenderTileComponent = Vue.component('CardRenderTileComponent',
           classes.push('card-tile-symbol-commercial-district');
         } else if (type === TileType.DEIMOS_DOWN) {
           classes.push('card-tile-symbol-deimos-down');
+        } else if (type === TileType.GREAT_DAM) {
```
Comment (kberg) [inline src/components/card/CardRenderTileComponent.ts:50]: Replace this growing list with a `Map<TileType, string>` Now or next time. In fact, do it on another PR on its own?

**Review by kberg — APPROVED** [review]
Minor comments only.

**Comment by kberg** [comment]
Feel free to merge when you are ready. I love it.

File: src/cards/base/GHGProducingBacteria.ts (hunk 1 above)
Comment (chosta) [reply src/cards/base/GHGProducingBacteria.ts:69]: Yes, I was considering this as a further refactoring. Probably doing it soon.

File: src/cards/base/GreatEscarpmentConsortium.ts (hunk 2 above)
Comment (chosta) [reply src/cards/base/GreatEscarpmentConsortium.ts:35]: I'm just copy/pasting from the existing cards. Not sure capital letters for resources is a good idea but we need to be consistent. I'll change it and will try to watch out for this in the future.

File: src/components/card/CardRenderTileComponent.ts (hunk 4 above)
Comment (chosta) [reply src/components/card/CardRenderTileComponent.ts:50]: I'm adding this to a list of minor refactoring TODOs on the render. I'll probably implement it as soon as I hit around 50% of all cards. At that point most of the logic + exceptions would be known so it might be a good time for an optimization iteration.
